docs/_posts/2017-02-11-warts.md.dev

```diff
@@ -9,6 +9,14 @@ order: 1
 Here is a list of built-in warts under the
 `org.wartremover.warts` package.
 
+### ArrayEquals
+
+Unlike other collections `==` on arrays and iterators checks reference equality:
```

Comment:
The wart is not about mutability. Both `mutable.Seq(1) == mutable.Seq(1)` and `immutable.Seq(1) == immutable.Seq(1)` bring the same result, I don't see the reason to specialize the description.